pac4j-kerberos/src/test/java/org/pac4j/kerberos/client/direct/DummyAuthenticator.java

```diff
@@ -0,0 +1,14 @@
+package org.pac4j.kerberos.client.direct;
+
+import org.pac4j.core.context.WebContext;
+import org.pac4j.core.credentials.Credentials;
+import org.pac4j.core.credentials.authenticator.Authenticator;
+import org.pac4j.core.exception.HttpAction;
+
+public class DummyAuthenticator implements Authenticator<Credentials> {
```

Comment:
I generally don't create this kind of class but use lambdas for brevity in tests. That said, it's not a big deal.